### PR TITLE
patch: update patch release schedule for February cycle

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

patch: update patch release schedule for February cycle

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

/assign @saschagrunert @hasheddan

### CODE REVIEW

File: releases/patch-releases.md [hunk 1]
```diff
@@ -47,31 +47,32 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Target date |
 | --- | --- |
-| January 2021 | 2021-01-13 |
```
Comment (xmudrii) [inline releases/patch-releases.md:50]: Should we add April instead of January, so we still have 3 items on the list?

**Review by saschagrunert — APPROVED** [review]
Other than @xmudrii's comment LGTM

File: releases/patch-releases.md (hunk 1 above)
Comment (cpanato) [reply releases/patch-releases.md:50]: will add more dates :)

Comment (cpanato) [reply releases/patch-releases.md:50]: done PTAL

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**